src/editor/nodes/FloorPlanNode.js

```diff
@@ -52,6 +60,20 @@ export default class FloorPlanNode extends EditorNodeMixin(FloorPlan) {
     node.maxTriangles = maxTriangles || 1000;
     node.forceTrimesh = forceTrimesh || false;
 
+    node._navMeshMode = navMeshMode || NavMeshMode.Automatic;
+
+    if (navMeshSrc) {
+      if (navMeshMode === NavMeshMode.Custom) {
+        loadAsync(
+          (async () => {
+            await node.load(navMeshSrc, onError);
```

Comment:
Do we need the wrapping function? I think `loadAsync(node.load(navMeshSrc, onError))`should do the same thing assuming `loadAsync` just expects a promise.